cmd/azbrowse/main.go

```diff
@@ -380,6 +485,32 @@ func main() {
 	}
 
 }
+func openEditor(filename string) error {
+	cmd := exec.Command("code", "--wait", filename) // TODO - make this configurable!
+	return cmd.Run()
+}
+func getAPIErrorMessage(responseString string) (string, error) {
+	var response map[string]interface{}
+
+	err := json.Unmarshal([]byte(responseString), &response)
+	if err != nil {
+		err = fmt.Errorf("Error parsing API response: %s: %s", responseString, err)
+		return "", err
+	}
+	if response["error"] != nil {
+		serializedError, err := json.Marshal(response["error"])
+		if err != nil {
+			err = fmt.Errorf("Error serializing error message: %s: %s", responseString, err)
+			return "", err
+		}
+		message := string(serializedError)
+		message = strings.Replace(message, "\r", "", -1)
+		message = strings.Replace(message, "\n", "", -1)
```

Comment:
Nice tweak for readability I guess?

Reply:
Yeah - the status section in the UI is too small to show the error when it contains line-breaks!